src/tests/Common/wasm-test-runner/WasmTestRunner.proj

```diff
@@ -7,6 +7,7 @@
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <BuildDir>$(MSBuildThisFileDirectory)\obj\$(Configuration)\wasm</BuildDir>
     <AppDir>$(TestBinDir)/WasmApp/</AppDir>
+    <NETCoreAppMaximumVersion>99.0</NETCoreAppMaximumVersion>
```

Comment:
Why do you need to set this here? Does this project not import the root's Directory.Build.targets which already sets this?

Reply:
This project runs in helix which is outside the repo, hence the common settings donot work

Reply:
How do other properties like `NetCoreAppCurrent` or `TestBinDir` work then?

Reply:
@safern  do u know why these properties work ?

Reply:
They are passed down globally: https://github.com/dotnet/runtime/blob/736b3042aa84de431838daf433de60dbaa645b4f/src/tests/Common/CLRTest.Execute.Bash.targets#L270